charts/fluent-operator/templates/fluentbit-fluentBit.yaml

```diff
@@ -18,6 +18,10 @@ spec:
   resources:
 {{- toYaml .Values.fluentbit.resources | nindent 4  }}
   fluentBitConfigName: fluent-bit-config
```

Comment:
The `fluentBitConfigName` and `namespaceFluentBitCfgSelector` are mutually exclusive?